shoop/admin/templates/shoop/admin/dashboard/dashboard.jinja

```diff
@@ -96,6 +96,9 @@
                 {% if activity %}
                     {% call regular_block("medium") %}{{ activity_block_content(activity) }}{% endcall %}
                 {% endif %}
+                <div class="block width-large">
+                    <p>{% trans %}Shoop version{% endtrans %}: {{ version }}</p>
```

Comment:
I don't think this is very translator friendly, since for some language it is possible that version number has to be before the text or between some words. Also, IMHO the word version is not needed in this context, it should be easy enough to understand that "3.0" in "Shoop 3.0" is a version number.

TLDR: Use:
```
<p>{{ _("Shoop %(version)s", version=version) }}</p>
```